src/main/java/com/alvarium/DefaultSdk.java

```diff
@@ -0,0 +1,25 @@
+package com.alvarium;
+
+import com.alvarium.utils.PropertyBag;
+
+public class DefaultSdk implements Sdk {
+
+  DefaultSdk() {
```

Comment:
Is the intent to add the other parameters required for the SDK's capabilities later?

https://github.com/project-alvarium/alvarium-sdk-go/blob/e82227181f2364a08214f3cfae33dd12610f607d/pkg/sdk.go#L37

Reply:
I'm going to assume the answer is "yes" pending approval of PR #34 given the logic from this PR that has been copied into that one.

Reply:
Yup, the SDK constructor will be implemented in a separate issue once we've implemented all of the dependencies.